prow.sh

```diff
@@ -987,6 +1047,16 @@ main () {
         if tests_need_non_alpha_cluster; then
             start_cluster || die "starting the non-alpha cluster failed"
 
+            # Install necessary snapshot CRDs and snapshot controller
+            # For Kubernetes 1.17+, we will install the CRDs and snapshot controller.
+            if version_gt "${CSI_PROW_KUBERNETES_VERSION}" "1.16.255" || "${CSI_PROW_KUBERNETES_VERSION}" == "latest"; then
```

Comment:
Is "1.16.255" 1.17?

Reply:
Yes, I couldn't find a better way to do version comparisons in bash. Open to suggestions on that